release-schema.json

```diff
@@ -1,73 +1,108 @@
-                        },
-                        "coordinates": {
-                            "title":"Coordinates",
-                            "description": "The relevant array of points, e.g. [latitude,longitude], or nested array, for the GeoJSON geometry being described. The longitude and latitude MUST be expressed in decimal degrees in the WGS84 (EPSG:4326) projection",
-                            "type": ["array","null"],
-                            "items": {"type": ["number","array"]}
-                        }
-                    }
-                },
-                "gazetteer": {
-                    "type": "object",
-                    "properties": {
-                        "scheme": {
-                            "title":"Gazetteer scheme",
-                            "description": "The entry of the selected gazetteer in the gazetteers codelist. The codelist provides details of services, where available, that can resolve a gazetteer entry to provide location names.",
-                            "type": ["string", "null"],
-                            "codelist":"locationGazeteers.csv",
-                            "openCodelist":true
-                        },
-                        "identifiers": {
-                            "title":"Identifiers",
-                            "description": "An array of one or more codes drawn from the gazetteer indicated in scheme.",
-                            "type": ["array", "null"],
-                            "items": {
-                                "type": ["string","null"]
-                            }
-                        }
-                    }
-                },
-                "uri": {
-                    "type":["string","null"],
-                    "title":"URI",
-                    "description":"A URI to a further description of the activity location. This may be a human readable document with information on the location, or a machine-readable description of the location."
-                }
+        "gazetteer": {
+          "type": "object",
+          "properties": {
+            "scheme": {
+              "title": "Gazetteer scheme",
+              "description": "The entry of the selected gazetteer in the gazetteers codelist. The codelist provides details of services, where available, that can resolve a gazetteer entry to provide location names.",
+              "type": [
+                "string",
+                "null"
+              ],
+              "codelist": "locationGazeteers.csv",
+              "openCodelist": true
+            },
+            "identifiers": {
+              "title": "Identifiers",
+              "description": "An array of one or more codes drawn from the gazetteer indicated in scheme.",
+              "type": [
+                "array",
+                "null"
+              ],
+              "items": {
+                "type": [
+                  "string",
+                  "null"
```

Comment:
Shouldn't all items be "string", if there are any items?

Reply:
Good point. With our current merge strategies, the array is what can be nulled, not the items.